Students/Iuga Cosmin/Lab5/src/pack/Customer.java

```diff
@@ -0,0 +1,62 @@
+package pack;
+
+import java.util.ArrayList;
+
+public class Customer {
+    private int id;
+    private String name;
+    private String contactInfo;
+    private ArrayList<BankAccount> bankAccounts;
+    public Customer(int id,String name,String contactInfo)
+    {
+        this.id=id;
+        this.name=name;
+        this.contactInfo=contactInfo;
+        this.bankAccounts=new ArrayList<>();
+    }
+    public void addBankAccount(BankAccount bankAccount)
+    {
+        bankAccounts.add(bankAccount);
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void makeTransaction(int id, BankAccount source, BankAccount destination, String type, double amount)
+    {
+        Transaction transaction = new Transaction(id,source,destination,type,amount);
```

Comment:
obiectul asta puteai sa-l faci in afara metodei si sa-l transmiti ca si parametru. asta in loc sa fie un carnat de X parametri.